suites/pacific/rbd/tier-1_rbd_mirroring_upgrade-4-to-latest.yaml

```diff
@@ -28,6 +28,8 @@ tests:
       module: install_prereq.py
       abort-on-fail: true
       desc: install ceph pre requisites
+      config:
```

Comment:
I think you set this config for testing purpose, Please revert it back.

Reply:
> The attached test execution link got failed, Please add successful test runs.

Failure is due to issue in ceph-ansible

Reply:
Unable to add successful runs at this point in time because of ceph-ansible issues. Will raise another PR to add changes if any once the ceph-ansible issues are resolved. @VasishtaShastry @sunilangadi2 hope that's okay?